src/pages/PostCompose.jsx

```
@@ -88,14 +88,28 @@ export default function PostCompose() {
+        }
+
+        // 3) If arriving with brand context (from Brand tab), add a selectable Brand community option for verified staff
+        try {
+          const sp = new URLSearchParams(location.search);
+          const ctxBrandId = sp.get('brandId');
+          const ctxBrand = sp.get('brand');
+          const isVerifiedStaff = hasRole && hasRole(['verified_staff', 'staff', 'brand_manager', 'super_admin']);
+          if (isVerifiedStaff && ctxBrandId && !items.some((c) => c.id === `brand-${ctxBrandId}`)) {
+            items.push({ id: `brand-${ctxBrandId}`, name: `Brand: ${ctxBrand || 'Brand'}`, isActive: true, isPublic: false, brandId: ctxBrandId, brandName: ctxBrand || 'Brand' });
+          }
         } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)